mail.py

```diff
@@ -0,0 +1,22 @@
+import smtplib, ssl
+
+
+
+class Mail:
+    def __init__(self, port, sender_email, sender_pw):
```

Comment:
all of the self attributes are static, so why do they need to be class attributes?